tests/e2e/portal-files/VTK_file.js

```diff
@@ -42,7 +42,7 @@ async function runTutorial () {
     await utils.takeScreenshot(page, screenshotPrefix + 'service_started');
 
     // Some time for setting up service's frontend
-    await tutorial.waitFor(3000);
+    await tutorial.waitFor(10000);
```

Comment:
A friendly reminder of how IMO we could handle a bit better the e2e timeouts heuristics please check this idea we already discussed: https://github.com/ITISFoundation/osparc-simcore/pull/6568#pullrequestreview-2381384201
Specially when the increase is due to the CI machines and not the system.

I am not trying to make a big  fuzz here :-) (actually I will approve this PR!) but rather keeping ideas flowing...